ext/app/electron/config.ts

```diff
@@ -0,0 +1,206 @@
+import * as path from 'path'
+import * as fse from 'fs-extra'
+import * as electron from 'electron';
+import * as ini from 'ini'
+import * as os from 'os'
+import * as log from 'app/server/lib/log';
+import * as packageJson from 'desktop.package.json';
+import {commonUrls} from 'app/common/gristUrls';
```

Comment:
(Would be great to make imports in alphabetical order, it is a convention followed by most but not all Grist code)